x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/create_signals.ts

```diff
@@ -108,6 +108,7 @@ export const createSignals = async ({
 
     return alertsClient.create({
       data: {
+        name: 'SIEM Alert',
```

Comment:
I am going to end up removing our name below from alert params and put it here for both the create and update since name looks like a first class citizen in the alerting world.

That is going to be a bit more involved, so I would say that just having this placeholder like you have it `SIEM Alert` is ok for this pull request.